sktime/base/_poly.py

```diff
@@ -0,0 +1,300 @@
+#!/usr/bin/env python3 -u
+# -*- coding: utf-8 -*-
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""Implements polymorphic base class."""
+
+__author__ = ["fkiraly", "miraep8"]
```

Comment:
No need to make me an author :)  Thanks for the thought though, I can add myself later if I contribute code.